framework/test/mocha/unit/modules/chain/submodules/delegates.js

```diff
@@ -140,45 +140,37 @@ describe('delegates', () => {
 		});
 
 		it('should update forging from enabled to disabled', async () => {
-			library.modules.accounts.getAccount(
-				{ publicKey: testDelegate.publicKey },
-				async (err, account) => {
-					expect(err).to.be.null;
-					expect(__private.keypairs[testDelegate.publicKey]).to.not.be
-						.undefined;
-					expect(account.publicKey).to.equal(testDelegate.publicKey);
-
-					const data = await library.modules.delegates.updateForgingStatus(
-						testDelegate.publicKey,
-						testDelegate.password,
-						false
-					);
+			const account = await library.components.storage.entities.Account.getOne({
+				publicKey: testDelegate.publicKey,
+			});
+			expect(__private.keypairs[testDelegate.publicKey]).to.not.be.undefined;
+			expect(account.publicKey).to.equal(testDelegate.publicKey);
 
-					expect(__private.keypairs[testDelegate.publicKey]).to.be.undefined;
-					expect(data.publicKey).to.equal(testDelegate.publicKey);
-				}
+			const data = await library.modules.delegates.updateForgingStatus(
```

Comment:
Having // Arrange // Act // Assert comments on top of this line would be really nice to differentiate which action is mutating data.

Arrange above line 143
Act above line 149
Assert above line 155

Reply:
This test will be rewritten soon, so lets skip this for now.